app/qml/AuthPanel.qml

```diff
@@ -0,0 +1,249 @@
+import QtQuick 2.7
+import QtQuick.Controls 2.0
+import QtQuick.Layouts 1.3
+import QtGraphicalEffects 1.0
+import QgsQuick 0.1 as QgsQuick
+
+Drawer {
```

Comment:
As shown on the screenshot, the auth panel does not work for me at all... There are some tweaks of z property and I don't understand the logic there. Why do we need to explicitly modify Z order? Maybe there's a better base Item than Drawer that would avoid the need to do those tweaks? Drawer is meant to be opened/closed by users, but what we actually want is just a simple Item that covers the whole screen...